cli/command/plugin/install.go

```diff
@@ -124,9 +124,8 @@ func buildPullConfig(ctx context.Context, dockerCli command.Cli, opts pluginOpti
 		Disabled:              opts.disable,
 		AcceptAllPermissions:  opts.grantPerms,
 		AcceptPermissionsFunc: acceptPrivileges(dockerCli, opts.remote),
-		// TODO: Rename PrivilegeFunc, it has nothing to do with privileges
```

Comment:
This no longer relevant? 😇

Reply:
It may be, but the comment should be on the API struct (thus in `moby/moby`)

Reply:
Why? This is referring to how to get auth credentials?

Reply:
So did you move this comment to moby/moby? 😄

Reply:
🤦‍♂️